code/obj/machinery/computer/cloning.dm

```diff
@@ -447,6 +447,11 @@ proc/eligible_to_clone(var/datum/mind/mind)
 	if (!M.client)
 		return null
 
+	if (istype(M, /mob/new_player))
+		var/mob/new_player/player = M
+		if (player.is_respawned_player)
+			return null
```

Comment:
feels kinda weird that as a `new_player` after respawning you keep the same mind as you had, but maybe that's needed for things :iiam:

Reply:
I think `mind` has ended up mostly just being used as "persistent `client`", although that gives it very little distinction from `player` so :iiam:x2